Analysis/bin/ShuffleMergeSpectral.cxx

```diff
@@ -724,6 +718,12 @@ class ShuffleMergeSpectral {
         std::cout << ".\n";
     }
 
+    static ROOT::ECompressionAlgorithm compAlg(const std::string& comp_string) {
+      if("ZLIB") return ROOT::kZLIB;
+      if("LZMA") return ROOT::kLZMA;
+      if("LZ4") return ROOT::kLZ4;
```

Comment:
The current implementation will result in undefined behaviour if `comp_string` is other than the considered 3 options.